### ISSUE TITLE
conv2d in mlp 

### ISSUE THREAD

**Issue by xinqiao123** [qwopqwop200/Neighborhood-Attention-Transformer#3]
Hello, 

Thanks for your great work, could I ask why we use conv2d layer in mlp module?  Thanks 

Xin 

class Mlp(nn.Module):
    def __init__(self, in_features, hidden_features=None, out_features=None, act_layer=nn.GELU, drop=0.):
        super().__init__()
        out_features = out_features or in_features
        hidden_features = hidden_features or in_features
        self.fc1 = nn.Conv2d(in_features, hidden_features,1)
        self.act = act_layer()
        self.fc2 = nn.Conv2d(hidden_features, out_features,1)
        self.drop = nn.Dropout(drop)

**Comment by qwopqwop200** [comment]
Use cnn instead of mlp to make the NAT take inputs (B,C,H,W) in the same shape as cnn.
Also, since cnn with a kener size of 1 operates the same as mlp, cnn was used.

**Comment by xinqiao123** [comment]
Thank you for your reply, It's really helpful,
